python/test/test_ingest_stream.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import pytest
+from timeplus import Stream, Query
+import datetime
+
+
+def test_ingest(test_environment, test_stream):
+    data = [["time", "data"], [[1, "efgh"]]]
+    try:
+        test_stream.ingest(*data)
+    except Exception as e:
+        pytest.fail(f"Ingest method failed with exception {e}")
+
+    test_stream.delete()
```

Comment:
should not delete the the test stream here

also, for all ingest test, you should verify if the data is in the stream now